docs/byte/1.md

```diff
@@ -344,26 +344,24 @@ $
 
 ## Questions and/or exercises
 
-As always, these questions are optional and only for fun.
```

Comment:
I'd probably leave this bit in. People shouldn't feel like they're missing out if they can't/don't want to take the time for these.